[BUG] 6.0.0 sdist lost `docs` folder, causing tests to fail

## Describe the bug
The 6.0.0 sdist (after switch to hatchling) no longer contains the `docs` folder. Besides losing the ability to build docs, this also causes tests to fail now.

## To Reproduce
```
$ wget -q https://files.pythonhosted.org/packages/af/ce/127d44302810184b1680ba5e0ab588325cf427d1a5e8c8479dd2cec80e97/icalendar-6.0.0.tar.gz
$ tar -xf icalendar-6.0.0.tar.gz
$ cd icalendar-6.0.0
$ PYTHONPATH=src pytest
/usr/lib/python3.12/site-packages/pytest_asyncio/plugin.py:208: PytestDeprecationWarning: The configuration option "asyncio_default_fixture_loop_scope" is unset.
The event loop scope for asynchronous fixtures will default to the fixture caching scope. Future versions of pytest-asyncio will default the loop scope for asynchronous fixtures to function scope. Set the default fixture loop scope explicitly in order to avoid unexpected behavior in the future. Valid fixture loop scopes are: "function", "class", "module", "package", "session"

  warnings.warn(PytestDeprecationWarning(_DEFAULT_FIXTURE_LOOP_SCOPE_UNSET))
========================================================= test session starts =========================================================
platform linux -- Python 3.12.6, pytest-8.3.3, pluggy-1.5.0
rootdir: /tmp/icalendar-6.0.0
configfile: pyproject.toml
testpaths: src/icalendar/tests
plugins: xdist-3.6.1, xprocess-1.0.2, subtests-0.13.1, import-check-0.0.4, asyncio-0.24.0, pkgcore-0.12.28, anyio-4.6.0
asyncio: mode=Mode.STRICT, default_loop_scope=None
collected 4049 items / 1 error                                                                                                        

=============================================================== ERRORS ================================================================
______________________________________ ERROR collecting src/icalendar/tests/test_with_doctest.py ______________________________________
src/icalendar/tests/test_with_doctest.py:57: in <module>
    for filename in os.listdir(os.path.join(DOCUMENTATION_PATH, subdir))
E   FileNotFoundError: [Errno 2] No such file or directory: '/tmp/icalendar-6.0.0/src/icalendar/tests/../../../docs'

During handling of the above exception, another exception occurred:
src/icalendar/tests/test_with_doctest.py:61: in <module>
    raise OSError("Could not find the documentation - remove the build folder and try again.")
E   OSError: Could not find the documentation - remove the build folder and try again.
======================================================= short test summary info =======================================================
ERROR src/icalendar/tests/test_with_doctest.py - OSError: Could not find the documentation - remove the build folder and try again.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================== 1 error in 0.75s ===========================================================
```

## Expected behavior
Passing tests (and ideally, the retained ability to build docs).

## Environment
 - [x] OS: Gentoo Linux amd64
 - [x] Python version: 3.12.6
 - [x] `icalendar` version: 6.0.0

## Additional context
- [x] I tested it with the latest version `pip3 install https://github.com/collective/icalendar.git`
- [x] I attached the ICS source file or there is no ICS source file

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -19,7 +19,7 @@ New features:
 Bug fixes:
 
 - Fix a few ``__all__`` variables.
-- Add missing ``docs`` folder to distribution packages, see `Issue 712 <https://github.com/collective/icalendar/issues/712>`_
+- Added missing ``docs`` folder to distribution packages. See `Issue 712 <https://github.com/collective/icalendar/issues/712>`_.
 
 6.0.0 (2024-09-28)
 ------------------

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -19,6 +19,7 @@ New features:
 Bug fixes:
 
 - Fix a few ``__all__`` variables.
+- Add missing ``docs`` folder to distribution packages, see `Issue 712 <https://github.com/collective/icalendar/issues/712>`_
 
 6.0.0 (2024-09-28)
 ------------------

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -93,7 +93,6 @@ exclude = [
   "/.*",
   "/*.*",
   "/src/icalendar/fuzzing",
-  "/docs",
   "/dist",
   "/build",
   "/htmlcov",

--- a/src/icalendar/tests/test_create_release.sh
+++ b/src/icalendar/tests/test_create_release.sh
@@ -24,7 +24,7 @@ if tar -tf "$archive" | grep -q 'fuzzing/'; then
 fi
 
 if ! tar -tf "$archive" | grep -q '/docs/'; then
-  echo "ERROR: The documentation is not included in the release."
+  echo "ERROR: The documentation is not included in the release, but should be."
   echo "       See https://github.com/collective/icalendar/issues/712"
   exit 1
 fi

--- a/src/icalendar/tests/test_create_release.sh
+++ b/src/icalendar/tests/test_create_release.sh
@@ -17,9 +17,15 @@ if ! [ -f "$archive" ]; then
   exit 1
 fi
 
-if tar -tf "$archive" | grep 'fuzzing/'; then
+if tar -tf "$archive" | grep -q 'fuzzing/'; then
   echo "ERROR: Fuzzing files are included in the release."
   echo "       See https://github.com/collective/icalendar/pull/569"
+  exit 1
+fi
+
+if ! tar -tf "$archive" | grep -q '/docs/'; then
+  echo "ERROR: The documentation is not included in the release."
+  echo "       See https://github.com/collective/icalendar/issues/712"
   exit 1
 fi
 

